### DISCUSSION TITLE
Add dynamodb store support for serverless lambda usage

### DISCUSSION THREAD

**Discussion by softmarshmallow** [nfriedly/express-rate-limit#421] [Ideas]
I'm providing cors.bridged.xyz service free to use, and I'me having some serious budget issues. 
https://github.com/bridgedxyz/base/issues/23

We're trying to add access limit with this library, but does not best fit to our circumstances. 

Having dynamodb support seems pretty standard and help AWS users to adopt this package a lot. (also with serverless usage)

:)

Share your thoughts!

**Comment by nfriedly** [comment]
Hi @softmarshmallow,

That sounds like a great idea, although it's probably not something I will personally do any time soon. 

If you'd like to, though, please feel free to make your own dynamodb store; you can use the existing stores as guides for how to set it up.

**Comment by softmarshmallow** [comment]
> Hi @softmarshmallow,
> 
> 
> 
> That sounds like a great idea, although it's probably not something I will personally do any time soon. 
> 
> 
> 
> If you'd like to, though, please feel free to make your own dynamodb store; you can use the existing stores as guides for how to set it up. 

Will do one my spare time ;)
